flight/PiOS/STM32F4xx/pios_flash_internal.c

```diff
@@ -34,6 +36,9 @@
 #include "pios_semaphore.h"
 #include <stdbool.h>
 
+/* Todo: consider relocating to a good header */
+bool __attribute__((weak)) vehicle_is_armed = false;
```

Comment:
OK, but this file isn't in many targets-- do we stub it out?  what if someone doesn't PIOS_INCLUDE_FLASH_INTERNAL etc